Index: ListViewDemoMamatha/app/src/main/java/com/house/listviewdemo/ListAdapter.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ListViewDemoMamatha/app/src/main/java/com/house/listviewdemo/ListAdapter.java	(revision ca31389cd8621b340196ab70b2fdb3f5aadd2d29)
+++ ListViewDemoMamatha/app/src/main/java/com/house/listviewdemo/ListAdapter.java	(revision 4c4e24d1c470cbae8cf06cedfc79733d5facf77e)
@@ -41,6 +41,16 @@
         return i;
     }
 
+    public void getItemVisiblity(){
+        {
+            System.out.print("hi");
+            System.out.print("hi");
+            System.out.print("hi");
+            System.out.print("hi");
+        }
+    }
+
+
     @Override
     public View getView(int i, View view, ViewGroup viewGroup) {
         ViewHolder holder;
@@ -49,12 +59,7 @@
             view = inflater.inflate(R.layout.row_layout, null);
             holder.textView = view.findViewById(R.id.textView);
             view.setTag(holder);
-            {
-                System.out.print("hi");
-                System.out.print("hi");
-                System.out.print("hi");
-                System.out.print("hi");
-            }
+
         } else {
             holder = (ViewHolder) view.getTag();
             holder.textView.setText("" + stringList.get(i));
